### PR TITLE
[14.0][MIG] l10n_be_national_number

### PR DESCRIPTION
**Pull request by carmenbianca**
internal task: https://gestion.coopiteasy.be/web#view_type=form&model=project.task&id=10633&active_id=10633&menu_id=

needed for https://github.com/OCA/cooperative/pull/69

### CODE REVIEW

File: l10n_be_national_number/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Copyright 2009-2019 Noviat.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    national_number = fields.Char(string="National Number")
```
Comment (carmenbianca) [inline l10n_be_national_number/models/res_partner.py]: Is this field even used? `id_numbers` appears to be the field that is _actually_ used. This field `national_number` is displayed in no view. I also can't find downstream usage of this field.

**Review by victor-champonnois — APPROVED** [review]
Wow that's weird, how could it have worked without the import statement !

File: l10n_be_national_number/models/res_partner.py (hunk 1 above)
Comment (victor-champonnois) [reply l10n_be_national_number/models/res_partner.py]: I think you are right ! I think I made a mistake.

Comment (victor-champonnois) [reply l10n_be_national_number/models/res_partner.py]: Maybe let's delete this module then ?

Comment (carmenbianca) [reply l10n_be_national_number/models/res_partner.py]: The only important thing in this module is the category definition in the data file :) We can remove this Python code.

File: l10n_be_national_number/data/res_partner_id_category.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+  <record id="l10n_be_national_number_category" model="res.partner.id_category">
+    <field name="code">l10n_be_national_number</field>
+    <field name="name">Belgium National Number</field>
```
Comment (huguesdk) [inline l10n_be_national_number/data/res_partner_id_category.xml:6]: shouldn’t this be “Belgian national registration number” (translated to “Numéro de registre national belge”)? or something like “Belgium: national registration number” or ”National registration number (Belgium)”? https://sma-help.bosa.belgium.be/en/faq/where-can-i-find-my-national-registration-number

**Review by huguesdk — COMMENTED** [review]
i think the official name for this identification number is “national registration number”. or is this category also used for the bis number? then the module should be called “l10n_be_identification_number” (https://sma-help.bosa.belgium.be/en/faq/i-do-not-know-my-belgian-identification-number).

**Review by huguesdk — APPROVED** [review]
this is a port of a module in 12.0 that is used in production since several months and that it’s too late too rename things at this stage before deployment, but we should plan a renaming at some point.

otherwise, lgtm!

**Comment by carmenbianca** [comment]
@victor-champonnois can you review/approve this?

**Review by remytms — APPROVED** [review]
LGTM

**Comment by victor-champonnois** [comment]
@carmenbianca Remy just approved so it's good to merge, you don't need my review anymore

**Comment by carmenbianca** [comment]
@robinkeunen please merge :)

**Pull request merged**